src/modules/Home/components/home-register/template.php

```diff
@@ -8,12 +8,12 @@
 ?>
 <div class="home-register">
     <div class="home-register__background">
-        <img class="home-register__background--img" src="<?php $this->asset($app->config['module.home']['home-register']) ?>" />
+        <img class="home-register__background--img" :src="subsite?.files?.signupBanner ? subsite?.files?.signupBanner?.url : '<?php $this->asset($app->config['module.home']['home-register']) ?>'" />
```

Comment:
Esta alteração só surgiu na v7.5.0, portanto a linha deve ser mantida inalterada conforme a tag [v7.4.26 do repositório original](https://github.com/mapasculturais/mapasculturais/blob/v7.4.26/src/modules/Home/components/home-entities/template.php).
```suggestion
        <img class="home-register__background--img" src="<?php $this->asset($app->config['module.home']['home-register']) ?>" />
```